position-node/src/worker.ts

```diff
@@ -0,0 +1,21 @@
+import { Worker } from '@temporalio/worker';
+
+async function run() {
+  // Worker connects to localhost by default and uses console error for logging.
+  // Customize the Worker by passing more options to create().
+  // create() tries to connect to the server and will throw if a connection could not be established.
+  // You may create multiple Workers in a single process in order to poll on multiple task queues.
+  // In order to configure the server connection parameters and other global options,
+  // use the Core.install() method to configure the Rust Core SDK singleton.
+  const worker = await Worker.create({
+    workflowsPath: `${__dirname}/workflows`,
+    nodeModulesPath: `${__dirname}/../node_modules`,
```

Comment:
```suggestion
    workDir: __dirname,
    activities: undefined,
```

@bergundy I tried this^ simpler suggestion, but it failed because of MODULE_NOT_FOUND activities. 

```
Error: Cannot find module '[...]/temporal-pendulum/position-node/src/activities'
```

Reply:
Then this is a bug, this shouldn't happen.